packages/microfrontend-sdk/README.md

```diff
@@ -0,0 +1,2 @@
+# Console SDK
```

Comment:
```suggestion
# Microfrontend SDK
```

Reply:
We should create an issue to add something to the various README

Reply:
https://github.com/mia-platform/console-sdk/issues/4